pages/search/index.vue

```diff
@@ -291,7 +297,21 @@
         this.rerouteSearch({ qf: this.qfForSelectedFacets, reusability: this.reusability, page: '1' });
       },
       selectView (view) {
+        if (process.browser) {
+          sessionStorage.view = view;
```

Comment:
As discussed in person, let's preempt the likely need to make the local/session storage key more specific, by creating a top-level `settings` object like:
```js
// in an init function somewhere:
sessionStorage.settings = sessionStorage.settings || {};
sessionStorage.settings.results = sessionStorage.settings.results || {};
sessionStorage.settings.results.view = sessionStorage.settings.results.view || 'grid';
```

Reply:
this won't work unless we also add a mechanism for parsing sessionStorage values as they only take 'simple' values. See https://vuejs.org/v2/cookbook/client-side-storage.html#Working-with-Complex-Values
I think adding that at this point is out of scope especially as we have only one thing to store there for now.

I have however renamed 'view' to 'searchResultsView' so as to make the name more specific and thereby less likely to interfere with any future use we may have for "sessionStorage.view"